Lib/test/test_exceptions.py

```diff
@@ -1336,6 +1337,26 @@ def test_unicode_errors_no_object(self):
         for klass in klasses:
             self.assertEqual(str(klass.__new__(klass)), "")
 
+    def test_unicode_error_str_gh_123378(self):
+        for formatter, start, end, obj in product(
+            (str, repr),
```

Comment:
Testing repr() if we already test str() sounds redundant to me. Is it really worth it?

Reply:
No you're right. I'll remove it.